src/Prometheus/Conduit.hs

```diff
@@ -48,3 +49,9 @@ prometheusSource PrometheusLocation { psPort = (Port port), psPath = (Path path)
 
 prometheusRenderer :: Monad m => ConduitT PrometheusLine ByteString m ()
 prometheusRenderer = map renderPrometheusLine .| map (<> "\n")
+
+identityRewriter :: Monad m => ConduitT PrometheusLine PrometheusLine m ()
+identityRewriter = map id
+
+addLabel :: Monad m => LabelName -> LabelValue -> ConduitT PrometheusLine PrometheusLine m ()
```

Comment:
If we end up with many of these rewriters it could be nice to have a newtype or alias once we have a few more.